### DISCUSSION TITLE
Waveshare 4inch Raspberry Pi LCD Display,480x320 Backlight problem

### DISCUSSION THREAD

**Discussion by siamkater** [Bodmer/TFT_eSPI#1817] [General]
Hello,
Please forgive me if this is a silly question, I'm a bloody beginner!

I have this Display Waveshare 4inch Raspberry Pi LCD Display,480x320 (https://www.waveshare.com/4inch-rpi-lcd-a.htm), and it is running with the Driver: RPI_ILI9486 on a ESP32
every thing is working fine, but I'm not able to dimm the Backlight. In the Data sheet (https://www.waveshare.com/w/upload/7/78/ILI9486_Datasheet.pdf) I found out that I have to use the Content Adaptive Brightness Control now my problem is that I didn't found the right command to write data to the register 
0x55 (Write Content Adaptive Brightness Control value data sheet, page 67)
Can someone point me in the right direction how I can solve this problem

greets

Andy

**Comment by Bodmer** [comment]
Although some chips have features to control the backlight LEDs these are rarely connected. The Waveshare display simply connects the backlight LEDs to the power source and thus the brightness is not controllable.
